Draft.md

```diff
@@ -156,235 +156,171 @@ Eine Personengruppe, die besonders oft zum passiven Spielball von Forschungssoft
 
 ## Mögliche Vorgehensweise zur Rezension archäologischer Forschungssoftware
 
-Nach unserem Verständnis sollte eine Rezension archäologischer Software, so wie es eine herkömmliche Besprechung einer wissenschaftlichen Publikation leistet, die Software zunächst vorstellen, dabei zusammengefasst Eckdaten vorlegen und den Kontext erläutern. Im Anschluss sollte eine kritische Beurteilung erfolgen.
+Nach unserem Verständnis sollte eine Rezension archäologischer Software, so wie es eine herkömmliche Besprechung einer wissenschaftlichen Publikation leistet, die Software zunächst vorstellen, dabei zusammengefasst Eckdaten vorlegen und den dann den ausführlicheren Kontext erläutern. Schließlich sollte eine kritische Beurteilung erfolgen.
 
-Umgesetzt auf Software gehören zu den Eckdaten Angaben zur Version und den Release-Daten, zu den Entwicklern, der Lizenz und Weiteres. Als Kontext verstehen wir eine erste Einordnung in das archäologische Forschungsfeld und Angaben zum möglichen Zusammenhang mit Forschungsprojekten, Arbeitsgruppen oder Institutionen. Für die kritische Beurteilung aus unterschiedlicher Perspektive haben wir unten einen Katalog an Fragen zusammengestellt. Zuletzt sollten die Ergebnisse der Betrachtung in einer Stellungnahme zusammengefasst werden, die die Software hinsichtlich ihrer Nützlichkeit, ihrer Bedienbarkeit, ihrer handwerklichen Qualität und ihrer Position in Relation zu den Idealen guter Forschungssoftware (z.B. FAIR und CARE), beurteilt.
+Umgesetzt auf Software gehören zu den Eckdaten Angaben zur Version, zu den Entwicklern, der Lizenz und Weiteres. Wir schlagen vor, die Rezension mit einer tabellierten Übersicht wie der folgenden einzuleiten:
 
-Selbstverständlich setzt die Rezension einer Forschungssoftware die Sichtung der Dokumentation, begleitender Publikationen ebenso voraus, wie die praktische Erprobung der Software selbst. Dabei ist eine realistische und transparente Einschätzung der eigenen Kompetenzen und des eigenen Nutzungsinteresses ein wichtiger Anhaltspunkt für die Leser, denn die Leserschaft ist wesentlich heterogener hinsichtlich ihrer Interessen, als man es bei der Besprechung einer Fachpublikation annehmen darf. Es ist eine wichtige Information, ob die Rezension ganz von der Anwendungsperspektive geschrieben ist, oder auch die Entwicklerperspektive umfasst.  
+- Name: ...
+- Kurzbeschreibung: ...
+- Rezensierte Version: ...
+- Plattform: ...
+- URLs und Entwicklerkontakt: ...
+- Publikationen: ...
+- Lizenzierung und Codeverfügbarkeit: ...
+- Alternativprodukte: ...
+
```

Comment:
- "Plattform" ist jedenfalls für mich Techie-Slang, ich bin mir z.B. nicht ganz sicher, ob ich weiß, was gemeint ist.
- URL und Entwicklerkontakt würde ich gerne trennen und URL zudem in einen menschensprachlichen Begriff überführen; "Entwicklerkontakt" ist vielfach nicht ganz klar anzugeben, denke ich. 
- Publikationen gehört für mich in eine Fußnote abgefrühstückt
- Was ist mit "Codeverfügbarkeit" gemeint? 
- Alternativprodukte? Gibt es das eigentlich so oft in der Archäologie ? bei der Software die ich besprochen habe gab es vielleicht Projekte, die ähnliche Probleme angefasst haben, aber Alternativ im Sinne, Word oder Libre Office Writer war das gar nicht.

Reply:
Habe in 3bff6d3 mal eine neue Version der Tabelle eingefügt.